python/ql/lib/semmle/python/essa/Essa.qll

```diff
@@ -501,7 +501,8 @@ class AssignmentDefinition extends EssaNodeDefinition {
   ControlFlowNode value;
 
   AssignmentDefinition() {
-    SsaSource::assignment_definition(this.getSourceVariable(), this.getDefiningNode(), value)
+    SsaSource::assignment_definition(this.getSourceVariable(), this.getDefiningNode(), value) and
+    not this instanceof ParameterDefinition
```

Comment:
Consider moving this restriction into `assignment_definition`.

Reply:
yes, that's better :+1: Done in 98ed5cf52277197024feec05908a8af01aa20ec8